src/maintaining/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Maintainer Guide"
+layout: default
+---
+
+Zinc products and services would devolve into absurdity without the efforts of
+our lovely _Maintainers_.
+
+Maintainers guide our products by facilitating conversations, soliciting and
+listening to feedback, breaking work down into pieces that Contributors can
+digest safely and easily, applying the changes submitted by Contributors.
+
+## Maintainer Ground Rules
+
+The keywords _must_, _must not_, and _should_ in this document are to be
+interpreted as described in [RFC 2119][rfc-2119].
+
+- Maintainers _must_ comply with Zinc's [code of conduct][code-of-conduct].
+- Maintainers _must not_ erase authorship information from any Contributors.
+- Maintainers _should_ ensure that all feedback provided to Contributors is
+  specific, actionable, and kind.
+- Maintainers _should_ respond to Contributors' issues, comments, and patches
+  within two weeks.
+
+[code-of-conduct]: https://www.zinc.coop/code-of-conduct/
+[rfc-2119]: https://tools.ietf.org/html/rfc2119
+
+## Launching a Product
+
+Any Maintainer may create a new project within Zinc at any time for any reason.
+In fact, we encourage Maintainers to sprout new projects to explore new problem
+domains, scratch an itch, decompose some functionality, or pretty much any
+reason.
+
+By creating space for everyone to explore and play in the manner they see fit,
+we'll have a more vibrant ecosystem of products and more entrance points for
+people to plug in and contribute their time, attention, and expertise.
+
+## First, Create a New Project on GitHub
+
+We encourage (but do not require) Maintainers to use our [New Project Template]
+when [creating a new project][create-a-project-on-github].
+
+It includes:
+
+1. Our Code of Conduct.
+1. A template Readme.
+1. Our standard License.
+1. The Contributing guide.
```

Comment:
Is the inconsistent case intentional or should these be all lower cases?
A template readme, Our standard License, The contributing guide.

Reply:
Probably should be consistent! Thank you :)